### PR TITLE
컬렉션 관리 기능에 추적 코드 붙이기

### PR DESCRIPTION
**Pull request by x86chi**
추적 코드 목록
- collection:create, `{ spaceId, collectionId, postIds?: string[] }`
- collection:update, `{ spaceId, collectionId, postIds?: string[] }`

postIds 프로퍼티가 있는 경우 목록을 수정하는 모달에 한해 추가가 됩니다.

### CODE REVIEW

**Comment by x86chi** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/penxle/penxle/930?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #928** <a href="https://app.graphite.dev/github/pr/penxle/penxle/928?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #929** <a href="https://app.graphite.dev/github/pr/penxle/penxle/929?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #930** <a href="https://app.graphite.dev/github/pr/penxle/penxle/930?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/penxle/penxle/930?utm_source=stack-comment).

File: apps/penxle.com/src/lib/components/pages/collections/CreateCollectionModal.svelte [hunk 1]
```diff
@@ -46,9 +47,10 @@
     initialValues: {
       collectionId: '',
     },
-    onSuccess: () => {
+    onSuccess: ({ id }) => {
       open = false;
       thumbnail = null;
+      mixpanel.track('collection:create', { spaceId, collectionId: id });
```
Comment (devunt) [inline apps/penxle.com/src/lib/components/pages/collections/CreateCollectionModal.svelte:53]: P1: `space:collection:create`

File: apps/penxle.com/src/lib/components/pages/collections/ManageCollectionModal.svelte [hunk 2]
```diff
@@ -87,8 +89,13 @@
     `),
     extra: () => ({ postIds: [...registeredPostIds.values()] }),
     schema: SetSpaceCollectionPostSchema,
-    onSuccess: () => {
+    onSuccess: ({ id }) => {
       open = false;
+      mixpanel.track('collection:update', {
```
Comment (devunt) [inline apps/penxle.com/src/lib/components/pages/collections/ManageCollectionModal.svelte:94]: P1: `space:collection:update`

File: apps/penxle.com/src/lib/components/pages/collections/UpdateCollectionModal.svelte [hunk 3]
```diff
@@ -47,8 +49,9 @@
     `),
     schema: UpdateSpaceCollectionSchema,
     extra: () => ({ thumbnailId: thumbnail?.id }),
-    onSuccess: () => {
+    onSuccess: ({ id }) => {
       open = false;
+      mixpanel.track('collection:update', { spaceId, collectionId: id });
```
Comment (devunt) [inline apps/penxle.com/src/lib/components/pages/collections/UpdateCollectionModal.svelte:54]: P1: `space:collection:update`

File: apps/penxle.com/src/lib/components/pages/posts/PostManageTable.svelte [hunk 4]
```diff
@@ -587,7 +587,14 @@
         const name = event.target.collectionName.value;
 
         const { id: collectionId } = await createSpaceCollection({ name, spaceId: selectedPosts[0].space.id });
-        await setSpaceCollectionPosts({ collectionId, postIds: selectedPosts.map((post) => post.id) });
+        const postIds = selectedPosts.map((post) => post.id);
+        await setSpaceCollectionPosts({ collectionId, postIds });
+
+        mixpanel.track('collection:create', {
```
Comment (devunt) [inline apps/penxle.com/src/lib/components/pages/posts/PostManageTable.svelte:593]: P1: `space:collection:create`

**Review by devunt — CHANGES_REQUESTED** [review]

**Review by devunt — APPROVED** [review]

**Pull request merged**